features/old/buyers/invitations/for_admins.feature

```diff
@@ -28,7 +28,8 @@ Feature: Buyer Account Invitations
   Scenario: Sending an invitation for a partner account
     When I navigate to the page of the invitations of the partner "lol cats"
     And I send an invitation to "new@lolcats.net"
-    Then "new@lolcats.net" should receive an invitation to account "lol cats"
+    Then I should see "Invitation was successfully created and will be sent soon"
```

Comment:
Adding a UI check to ensure the request has been completed

Reply:
That's fine. But it's counter-intuitive to have to add a step not because it's required or make sense, but because it's a workaround to make Capybara wait. This probably means we'll keep creating flaky scenarios in the future.

Reply:
Yeah, would be good to have a list of common offenders to remind ourselves.

In general the step `"new@lolcats.net" should receive an invitation to account "lol cats"` can easily wait for this to happen. Like try multiple times waiting. But I have also felt overwhelmed with all kinds of async scenarios where checks can fail. So for now it is good enough for me.